pages/content/migration-guidance.mdx

```diff
@@ -459,6 +459,32 @@ We recommend you make a small number of attempts to export a dump of your databa
 
 If you think that your service will face issues migrating data using the documented approaches you should [get in contact with the team](https://admin.london.cloud.service.gov.uk/support/contact-us) using our support form to discuss your needs further. 
 
+## PaaS CDN Migration
+
+If you are migrating from a Cloud Foundry CDN route, be aware that this is implemented as an AWS Cloudfront distribution behind the scenes.
+
+This can create an issue if you are migrating to a different Cloudfront distribution. You will receive a 'CNAMEAlreadyExists' error if you try to set up the new Cloudfront distribution before the DNS cutover.
+
+AWS documents this issue on the following (page)[https://aws.amazon.com/premiumsupport/knowledge-center/resolve-cnamealreadyexists-error/].
+
+This means you will need to remove the domain from your cdn-route service in Cloud Foundry before you can add it to your new AWS Cloudfront distribution.
+
+This can be acheived in Cloud Foundry by removing the cdn-route service. E.g.
```

Comment:
I'm rejigging to make it clearer (I hope). We try not to use passive voice
```suggestion
We implement Cloud Foundry CDN routes as an AWS Cloudfront distribution behind the scenes.

If you are migrating from a Cloud Foundry CDN route, you will need to remove the domain from your cdn-route service before you can add it to your new AWS Cloudfront distribution. If you don't do this, you will receive a 'CNAMEAlreadyExists' error. (AWS documents this issue)[https://aws.amazon.com/premiumsupport/knowledge-center/resolve-cnamealreadyexists-error/].

You can either remove the cdn-route service:
```

Reply:
Updated. Github didn't let me commit your suggestion for some reason.